doc/user/download_backup.md

```diff
@@ -0,0 +1,49 @@
+# Download Backup
+
+The backups that etcd operator saves are [etcd snapshots](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/recovery.md).
+This document talks about the ways to download these backup/snapshot files.
+
+## Backup Service
+
+If etcd cluster with backup is still running, there is a backup service.
+It's named as `${CLUSTER_NAME}-backup-sidecar`. For example:
+```
+$ kubectl get svc
+etcd-cluster-backup-sidecar   10.39.243.229   <none>        19999/TCP           4m
+```
+
+Given the etcd version of the cluster, you can download backup via HTTP endpoint
+`http://${CLUSTER_NAME}-backup-sidecar:19999/backup?etcdVersion=${ETCD_VERSION}`. For example:
+```
+$ curl "http://etcd-cluster-backup-sidecar:19999/backup?etcdVersion=v3.1.0" -o v3.1.0_etcd.backup
```

Comment:
i created an issue to track this https://github.com/coreos/etcd-operator/issues/673